artifactory/commands/transferfiles/delayedartifactshandler.go

```diff
@@ -216,3 +326,70 @@ func createdDelayedArtifactsChannelMng() DelayedArtifactsChannelMng {
 	channel := make(chan FileRepresentation, fileWritersChannelSize)
 	return DelayedArtifactsChannelMng{channel: channel}
 }
+
+type SplitContentWriter struct {
```

Comment:
Let's add a description for this struct. Explain why it is needed.
Let's also describe all of the struct's components.
Also, it looks like this struct doesn't have have to be public.